scripts/mkdist.js

```diff
@@ -14,43 +14,40 @@ function mkdist (opts, cb) {
 
   var dist = opts.dist
   var ver = dist.version
-  if (ver.match(/^v/)) ver = ver.substring(1)
```

Comment:
@haadcode can you give some reasoning on that? How are people supposed to be able to differentiate builds, and updates on dist.ipfs.io if the version only says master?